### DISCUSSION TITLE
question about exercise 5.16 in the book (p.179)

### DISCUSSION THREAD

**Discussion by jie-huangfu** [EricDarve/numerical_linear_algebra#246] [General]
Is there a quick explanation for the exercise 5.16 (p. 179 in the book)? I'm having a hard time proving it. Or is there a reference?

**Comment by EricDarve** [comment]
We don't have a solution set. The key steps in the proof are showing that the first n-1 columns are linearly independent. This comes from the unreduced upper Hessenberg shape of the matrix. Assume that there is a non-trivial vector in the null space of the first n-1 columns, and you will reach a contradiction. Since the matrix is singular, the last column must be a linear combination of the previous n-1 columns. This implies that the last row of R is 0. Hope this helps.
